pyclean/modern.py

```diff
@@ -400,3 +404,101 @@ def suggest_debris_option(args):
             'Hint: Use --debris to also clean up build artifacts '
             'from common Python development tools.',
         )
+
+
+def build_git_clean_command(ignore_patterns, dry_run=False, force=False):
+    """
+    Build the git clean command with appropriate flags.
+
+    Args:
+        ignore_patterns: List of patterns to exclude from git clean
+        dry_run: If True, only show what would be deleted
+        force: If True, delete without prompting (otherwise interactive)
+
+    Returns:
+        List of command arguments
+    """
+    cmd = ['git', 'clean', '-dx']
+
+    for pattern in ignore_patterns:
+        cmd.extend(['-e', pattern])
```

Comment:
This won't work. `git clean` requires a single name after `-e`, hence `-e` has to be repeated for each pattern.

If possible, come up with a sweet one-liner. Is there maybe a nice built-in function we can use for that?